clarify options example file names (#5683)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/options/index.md

</html>
```

> Below is an example options script. Save it as `options.js` in the same folder as `options.html`. It lets you save a user's preferred options across devices by using the [storage.sync][1] API.

Reviewer: ```suggestion
Below is an example options script. Save it as `options.js` in the same folder as `options.html`. This saves the user's preferred options across devices using the [storage.sync][1] API.
```
